GameData/RP-1/Contracts/Earth Observation 1/EOSIRRad1.cfg

```diff
@@ -117,3 +118,258 @@ CONTRACT_TYPE
 		}
 	}
 }
+
+CONTRACT_TYPE

```

Comment:
Thoughts on doing 2 follow-on contracts like this and dumping the repeatable one?  Gets rid of the "launch more satellites for reasons" per one of your other comments, adds inclination to get more science for each one, and lowers rep so that the science is the goal.   Also mirrors the real launches since TIROS-9 was polar.

Reply:
Yep I think it makes sense to swap to fixed extras rather than infinite repeatables. Regarding rewards, generally optionals should have higher rep rewards than requireds, and my point about science as the goal was more that science was the goal _of the program_, rather than the goal of the player (who doesn't need a contract to gather science). Having science be the focus of the program contrasts with, say, CA1 (or even Crewed Orbit), where the goal is to Do Stuff In Space and science is at best a byproduct.